src/lib/admin/EditDocumentChunks.svelte

```
@@ -163,17 +162,20 @@
+			client
+				.mutation(api.pdfJobs.createJob, {
+					documentId: currentDocView as Id<'contentLib'>,
+					pdfUrl,
+					fileKey: fileKey || undefined
+				})
+				.then(() => {
+					params.delete('processing');
+					params.delete('pdfUrl');
+					params.delete('fileKey');
+					const newUrl = `${window.location.pathname}${params.toString() ? `?${params.toString()}` : ''}`;
+					replaceState(newUrl, {});
```

ESLint

[error] Unexpected replaceState() call without resolve().

(svelte/no-navigation-without-resolve)